actionpack/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+*   `ActionDispatch::Routing::Mapper::Resources::Resource#initialize` now raises
+    if the custom param option contains a colon (`:`)
```

Comment:
Let's write the changelog entry in a way everyday Rails users will find relevant. The PR summary would fit well here.